syntax = "proto3";

message SettingsMsg {
  string trap_name = 1;
  string wifi_ssid = 2;
  string wifi_password = 3;
  bool wifi_enabled = 4;
  int32 max_sessions = 5;
  float min_score = 6;
}